WPILib.Extras/SubsystemCommand.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using WPILib.Commands;
+
+namespace WPILib.Extras
+{
+    /// <summary>
+    /// A <see cref="ActionCommand"/> will run an <see cref="Action"/> when it is
```

Comment:
So, apart from fixing the docs...

I found this class to be very helpful when playing with WPILib in F#. It might seem hacky, but I personally feel that it's actually more hacky to have the Requires as a separate step. Example:

```fsharp
type Drive() =
    inherit Subsystem()
    // remaining setup, as needed
    override drive.InitDefaultCommand() =
        let cmd = {
            new SubsystemCommand(drive, "TurnZero") with
            override cmd.Initialize() = cmd.m_subsystem.ResetGyro()
            override cmd.Execute() = cmd.m_subsystem.TurnAngle(0.0)
            override cmd.IsFinished() = Math.Abs(cmd.m_subsystem.AngleError) < 0.1
            override cmd.End() = cmd.m_subsystem.StopMotors()
            override cmd.Interrupted() = cmd.End()
            }
        drive.SetDefaultCommand(cmd)
```

Admittedly, in my actual practice, instead of `cmd.m_subsystem` I used `drive` because the scoping rules are a bit different, but adding the Subsystem reference as `m_subsystem` was an afterthought from when I was porting it back to C#